AGENTS.md

```
@@ -1,224 +1,436 @@
+yarn tsc -p tsconfig.json --composite false --declaration false --declarationMap false --noEmit
+```
 
-# React SDK
-yarn workspace @stream-io/feeds-react-sdk run build
+## Architecture: core concepts
 
-# React Native SDK
-yarn workspace @stream-io/feeds-react-native-sdk run build
+### Package hierarchy
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)